cmake/compiler.cmake

```diff
@@ -63,8 +63,10 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
             CMAKE_CXX_FLAGS CMAKE_CXX_FLAGS_DEBUG CMAKE_CXX_FLAGS_RELEASE)
 #      if (DEBUG)
 #      set(${flag_var} "${${flag_var}} /MDd /nologo")
+#      set( CMAKE_CONFIGURATION_TYPES "DEBUG")
 #      else()
       set(${flag_var} "${${flag_var}} /MD /nologo")
+      set( CMAKE_CONFIGURATION_TYPES "Release")
```

Comment:
This will be set several times since it's in the middle of a for loop. Setting the configuration type doesn't really belong in the static CMake files either. This should be something configurable by the user at the start.